cmd/av/pr.go

```diff
@@ -18,15 +19,63 @@ var prCreateCmd = &cobra.Command{
 	Use:   "create",
 	Short: "create a pull request for the current branch",
 	RunE: func(cmd *cobra.Command, args []string) error {
-		if config.GitHub.Token == "" {
-			// TODO: lets include a documentation link here
-			logrus.Info(
-				"GitHub token is not configured. " +
-					"Consider adding it to your config file (at ~/.config/av/config.yaml) " +
-					"to allow av to automatically create pull requests.",
-			)
+		// Lets keep this commented until we actually implement this :')
+		//if config.GitHub.Token == "" {
+		//	// TODO: lets include a documentation link here
+		//	logrus.Info(
+		//		"GitHub token is not configured. " +
+		//			"Consider adding it to your config file (at ~/.config/av/config.yaml) " +
+		//			"to allow av to automatically create pull requests.",
+		//	)
+		//}
+
+		repo, err := getRepo()
+		if err != nil {
+			return errors.WrapIf(err, "failed to get repo")
+		}
+		currentBranch, err := repo.CurrentBranchName()
+		if err != nil {
+			return errors.WrapIf(err, "failed to determine current branch")
+		}
+		origin, err := repo.Origin()
+		if err != nil {
+			return errors.WrapIf(err, "failed to determine origin")
+		}
+
+		// figure this out based on whether or not we're on a stacked branch
+		var prBaseBranch string
+
+		stackBranch, err := stacks.GetBranch(repo, currentBranch)
```

Comment:
prefer: GetBranchMeta. It's not quite clear otherwise why are you passing the branch and getting branch back. Could also be `GetBranchInfo`